src/main/java/uk/gov/hmcts/reform/coh/controller/DeadlineController.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.coh.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.coh.domain.OnlineHearing;
+import uk.gov.hmcts.reform.coh.service.OnlineHearingService;
+import uk.gov.hmcts.reform.coh.service.QuestionService;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/continuous-online-hearings/{onlineHearingId}")
+public class DeadlineController {
+
+    private static final Logger log = LoggerFactory.getLogger(DeadlineController.class);
+
+    private OnlineHearingService onlineHearingService;
+    private QuestionService questionService;
+
+    @Autowired
+    public DeadlineController(
+        OnlineHearingService onlineHearingService,
+        QuestionService questionService
+    ) {
+        this.onlineHearingService = onlineHearingService;
+        this.questionService = questionService;
+    }
+
+    @PutMapping("/deadline-extensions")
+    public ResponseEntity requestExtensionForQuestion(@PathVariable UUID onlineHearingId) {
+        OnlineHearing onlineHearing = new OnlineHearing();
+        onlineHearing.setOnlineHearingId(onlineHearingId);
+        Optional<OnlineHearing> optionalOnlineHearing = onlineHearingService.retrieveOnlineHearing(onlineHearing);
+
+        if (!optionalOnlineHearing.isPresent()) {
+            return ResponseEntity.status(HttpStatus.NOT_FOUND).body("Online hearing not found");
+        }
+
+        try {
+            questionService.requestDeadlineExtension(optionalOnlineHearing.get());
```

Comment:
This is problematic. There's no feedback to the caller about what happens at all.

Reply:
The feedback is either success or a failure. Would a message look better?

Reply:
Ah, this was because of the run time exception. Once that's removed, then there'll be feedback.